.github/workflows/ci.yml

```diff
@@ -16,24 +16,25 @@ jobs:
       fail-fast: false
       matrix:
         suite:
-          - bundler
-          - cargo
-          - common
-          - composer
-          - dep
-          - docker
-          - elm
-          - git_submodules
-          - github_actions
-          - go_modules
-          - gradle
-          - hex
-          - maven
-          - npm_and_yarn
-          - nuget
-          - omnibus
-          - python
-          - terraform
+        - { path: bundler, name: bundler1 }
+        - { path: bundler, name: bundler2 }
```

Comment:
@brrygrdn @jurre looks like you can add hashes as the suite argument, thoughts on doing something like this?

Reply:
Yep, I dig it!

Reply:
🎉 Nice

Reply:
Ah, the only downside of this approach is it seems to mess with our configured required builds, we'll need to flip the required builds from the old set to the new ones, merge this and then force all other PRs to merge master.

Not a big deal overall, as I think this is ultimately a better approach, just a one-time pain to switch over.

Reply:
Good catch! Will fix that up 👍